packages/app/src/hooks/useChartConfig.tsx

```diff
@@ -13,6 +15,49 @@ import { IS_MTVIEWS_ENABLED } from '@/config';
 import { buildMTViewSelectQuery } from '@/hdxMTViews';
 import { getMetadata } from '@/metadata';
 
+export const isMetric = (config: ChartConfigWithOptDateRange) =>
+  config.metricTables != null;
+
+// TODO: apply this to all chart configs
```

Comment:
why do we want to overwrite all aliases?

Reply:
mainly for the metric chart since its select representation is not readable